### DISCUSSION TITLE
I can not update file

### DISCUSSION THREAD

**Discussion by Ashenafi-Abebe** [iterate-ch/cyberduck#15992] [Q&A]
This ticket system is to report bugs and feature requests. For support, visit the [help page](https://cyberduck.io/help/) first.

 * If you have trouble connecting to a server or your login credentials are not valid, try to resolve the issue with the assistance of your hosting service provider. Any such ticket will be closed with a resolution of _thirdparty_.

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. macOS]

**Log Files**
Please include the [log file](https://docs.cyberduck.io/cyberduck/support/#logging-output). You can reach the logging output in _Preferences → Connection_.

**Additional context**

**Comment by tatze96** [comment]
Please add the following information:
- Operating system in use
- Cyberduck version in use
- affected protocol
- Screenshot of the error message

**Sidenote**: Ensure to attach the screenshot through the GitHub web interface, as it wouldn't be visible to us otherwise.
